tests/functional/test_check.py

```diff
@@ -292,3 +294,23 @@ def test_check_include_work_dir_pkg(script):
     )
     assert matches_expected_lines(result.stdout, expected_lines)
     assert result.returncode == 1
+
+
+@pytest.mark.xfail(
+    reason='Not yet implemented: https://github.com/pypa/pip/pull/8633',
+    strict=True,
+)
+def test_check_integrity_errors_on_missing_files(data, script, tmpdir):
```

Comment:
By the way, this is a negative test. It's probably reasonable to add a positive test too (for when the integrity is fine)